Assets/MixedRealityToolkit.SDK/Experimental/Features/Dwell/DwellProfile.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Dwell
+{
+    [MixedRealityServiceProfile(typeof(DwellProfile))]
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Dwell Profile", fileName = "DwellProfile", order = 100)]
+    [Serializable]
+    public class DwellProfile : ScriptableObject
+    {
+        [Tooltip("Pointer type to use for triggering a dwell interaction")]
+        [SerializeField]
+        private InputSourceType dwellTriggerPointerType = InputSourceType.Head;
+
+        [Tooltip("Delay in seconds until it is determined that the user intends to interact with the target.")]
+        [SerializeField]
+        [Range(0, 2)]
+        private float dwellIntentDelay = 0;
+
+        [Tooltip("Delay in seconds until dwell feedback is started to be shown.")]
+        [SerializeField]
+        [Range(0, 5)]
+        private float dwellStartDelay = 3;
+
+        [Tooltip("Additional time (not including the timeToTriggerDwellInSec) the user needs to keep looking at the UI to trigger select on it.")]
+        [SerializeField]
+        [Range(0, 20)]
+        private float timeToCompleteDwell = 6;
+
+        [Tooltip("Should the system allow for dwell to resume if the pointer exits the target briefly.")]
+        [SerializeField]
+        private bool allowDwellResume = false;
+
+        [Tooltip("Time in seconds when gaze can fall off the target and come back.")]
+        [SerializeField]
+        [Range(0, 20)]
+        private float timeToAllowDwellResume = 1;
+
+        public InputSourceType DwellPointerType
```

Comment:
I think there should be a period of time after dwell completion where new dwells are not allowed on the same item.  Is this accounted for anywhere?  Or maybe it's just that as currently coded, you have to focus off then back on again to start a new dwell after completing one?  Maybe that's fine.

Reply:
Yes you have to focus of and then focus back on the target to trigger a fresh dwell.